scripts/copy_sample_cram_to_release.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+
+
+"""
+Given a project and sample IDs, copies cram files for
+each sample listed into the project's release bucket.
+"""
+
+import logging
+import sys
+import subprocess
+import click
+
+# pylint: disable=E0401,E0611
+from sample_metadata.apis import AnalysisApi
+from sample_metadata.models import AnalysisType
+
+
+def copy_to_release(project: str, path: str):
+    """
+    Copy a single file from a main bucket path to the equivalent release bucket
+    """
+    release_path = path.replace(
+        f'cpg-{project}-main',
+        f'cpg-{project}-release',
+    )
+
+    subprocess.run(f'gsutil cp {path} {release_path}', shell=True, check=True)
+    logging.info(f'Copied {release_path}')
+
+
+@click.command()
+@click.option('--project', '-p', help='Metamist name of the project', required=True)
```

Comment:
You can pull this from the analysis-runner config (you could leave this to provide an optional override), eg:

```
from cpg_utils.config import get_config

# this is set by the analysis-runner
config = get_config()

project = config['workflow']['dataset']
```